src/Sylius/Bundle/AdminBundle/Resources/config/services/twig/component.xml

```diff
@@ -134,6 +134,19 @@
             />
         </service>
 
+        <service
+            id="sylius_admin.twig.component.render_resource"
+            class="Sylius\Bundle\AdminBundle\Twig\Component\Shared\RenderEntityComponent"
```

Comment:
You have used two different names: `render_resource` and `renderEntity`, but neither name clearly indicates what these classes do. My suggestion is `renderTemplateWithResource`, though I'm not sure if it is a better name.